client/src/components/JurisdictionAdmin/JurisdictionAdminView.tsx

```diff
@@ -77,6 +78,14 @@ const JurisdictionAdminView: React.FC = () => {
     j => j.id === jurisdictionId
   )!
 
+  const isManifestUploaded = isFileProcessed(ballotManifest)
+  const isBatchTalliesUploaded =
+    !isBatchComparison || isFileProcessed(batchTallies!)
+  const isCvrsUploaded =
```

Comment:
nit: `areCvrsUploaded` might be more readable

Reply:
Since they are all referring to a single file upload, I'll leave for consistency. Though I agree it's confusing